test/torch/tensors/test_poly.py

```diff
@@ -0,0 +1,60 @@
+import syft 
+from syft.frameworks.torch.tensors.Polynomial import PolynomialTensor
+
+import math
+
+class TestPolynomialTensor:
+
+    """ Test cases to ensure working of Polynomial Tensor. The tests under these ensure that the error between actual funcion values and approximations do not deviate too much"""
+
+    def __init__(self):
+
+        self.tensor = PolynomialTensor()
+        # Maximum permissible error as calculated by EvalRelative under PolynomialTensor
+        self.threshold = 0.1
```

Comment:
The error of polynomial approximations grows without bound the further you go from zero. It seems a bit missleading to say we have a maximum permissible error when that is not the case